ServerStatisticsCollection/Program.cs

```diff
@@ -0,0 +1,32 @@
+﻿using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using RabbitMQClientLibrary.Configuration;
+using RabbitMQClientLibrary.MessagePublishingService;
+using ServerStatisticsCollection.Models.ConfigModels;
+using ServerStatisticsCollection.Services;
+
+var configuration = new ConfigurationBuilder()
```

Comment:
if you kept the appsettings.json on the root directory for the project you won't need this I guess